tests/datavolume_test.go

```diff
@@ -1681,6 +1681,73 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			By("Restore the profile")
 			updateStorageProfileSpec(f.CrClient, defaultScName, *originalProfileSpec)
 		})
+		It("[test_id:8170]Import succeeds when storage class is not specified, but access mode is", func() {
+			defaultScName := utils.DefaultStorageClass.GetName()
+
+			By(fmt.Sprintf("configure storage profile %s", defaultScName))
```

Comment:
I would put the piece of code that saves the storage profile in a BeforeEach, and the piece that restores it in an AfterEach

Reply:
done